app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
Giải thích toán tử: `||=`

Reply:
`||=` sẽ assign toán hạng bên trái với giá trị của chính nó `or` với giá trị trả về của phương thức `find_by`. Nếu `find_by` không tìm ra kết quả thì giá trị của `@current_user` này là `nil` và nếu tìm ra kết quả thì giá trị là object match kết quả đó